core/src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdStage.scala

```diff
@@ -252,8 +252,9 @@ import com.datastax.driver.core.utils.Bytes
       }
 
       override protected def onTimer(timerKey: Any): Unit = timerKey match {
-        case Continue            => continue()
-        case LookForMissingSeqNr => lookForMissingSeqNr()
+        case Continue if lookingForMissingSeqNr.isDefined => // regular ticks disabled when looking for missing seqNr
```

Comment:
I'd prefer to move that inside `continue()`, in case it's called or will be called from elsewhere